nipype/interfaces/fsl/preprocess.py

```diff
@@ -592,6 +592,62 @@ class ApplyXfm(FLIRT):
     input_spec = ApplyXfmInputSpec
 
 
+class ApplyXfm4DInputSpec(FSLCommandInputSpec):
+    in_file = File(exists=True, position=0, argstr='%s', mandatory=True, desc="timeseries to motion-correct")
+    ref_vol = File(exists=True, position=1, argstr='%s', mandatory=True, desc="volume with final FOV and resolution")
+    out_file = File(exists=True, position=2, argstr='%s', genfile=True, desc="file to write", hash_files=False)
```

Comment:
i think this one shouldn't have `exists=True` and should also have `hash_files=False` so that `out_file` doesn't get hashed if it exists.

i think the logic of this interface can be simplified greatly by using `name_source` and `name_template`

http://nipype.readthedocs.org/en/latest/devel/cmd_interface_devel.html#creating-outputs-on-the-fly